src/ffi/import.rs

```diff
@@ -82,3 +84,68 @@ extern "C" {
         initfunc: Option<extern "C" fn() -> *mut PyObject>,
     ) -> c_int;
 }
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+#[cfg(not(Py_LIMITED_API))]
+pub struct _inittab {
+    pub name: *mut c_char,
+    pub initfunc: Option<unsafe extern "C" fn()>,
+}
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+#[cfg(not(Py_LIMITED_API))]
+pub struct _frozen {
+    pub name: *const c_char,
+    pub code: *const c_uchar,
+    pub size: c_int,
+}
+
+#[cfg(not(Py_LIMITED_API))]
+extern "C" {
```

Comment:
This extern block contains `pub static mut` so it's going to need `#[cfg_attr(windows, link(name = "pythonXY"))]`